### ISSUE TITLE
Tests failing following semver 7.5.0 update

### ISSUE THREAD

**Issue by Steve-Mcl** [FlowFuse/flowfuse#2077]
### Current Behavior

We currently use `semver` on vue pages and since it was updated to v7.5.0 tests fail with the error: 
```
Module not found: Error: Can't resolve 'util' in '/home/runner/work/flowforge/flowforge/node_modules/semver/classes'
```

see [this failed run](https://github.com/flowforge/flowforge/actions/runs/4863637928/jobs/8671603797?pr=2042#step:5:38) for more detail.

<br>

Our current dependency of semver is `^7.3.8` meaning GH will use `V7.x` and thus hit [this issue](https://github.com/npm/node-semver/issues/554)

Possible workarounds include: (from best/hardest to worst/easiest)
* drop / replace webpack (something I believe @Pezmc is aiming for?)
* add a [fallback ](https://stackoverflow.com/questions/64402821/module-not-found-error-cant-resolve-util-in-webpack)in webpack.config.js 
* replace semver with a browser ready module
* browserfy semver
* lock semver to pre v7.5.0  - done in #2078 
   * until semver resolves this or makes a browser ready dist.

#### NOTE: 

Anyone installing FF fresh will probably also hit this issue

### Expected Behavior

tests to run without this error

### Steps To Reproduce

raise a new PR that causes frontend tests to run

### Environment

- FlowForge version: 1.6.x-git

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by knolleary** [comment]
Given we only use semver in one place in the frontend (introduced by #2042) another option is to just add the couple lines of code needed to do the required version check rather than import a whole module (whether semver or another) to do it.

This issue should not be the driving factor for replacing webpack - that's a bigger piece of work.

**Comment by Steve-Mcl** [comment]
Possible contender: https://www.npmjs.com/package/compare-versions

This NPM lib has a healthy weekly download, is half the size of semver, is browser ready and has no deps.

**Comment by Pezmc** [comment]
PR to follow is https://github.com/npm/node-semver/pull/559/files

**Comment by joepavitt** [comment]
Closed by [Update semver to avoid CVE #2718](https://github.com/FlowFuse/flowfuse/pull/2718)
